src/DssProxyActionsStETH.sol

```diff
@@ -0,0 +1,481 @@
+    }
+
+    function _getDrawDart(
+        address jug,
+        address urn,
+        bytes32 ilk,
+        uint256 wad
+    ) internal returns (int256 dart) {
+        // Updates stability fee rate
+        uint256 rate = JugLike(jug).drip(ilk);
+
+        // Gets DAI balance of the urn in the vat
+        uint256 dai = vat.dai(urn);
+
+        // If there was already enough DAI in the vat balance, just exits it without adding more debt
+        uint256 rad = _mul(wad, RAY);
+        if (dai < rad) {
+            uint256 toDraw = rad - dai; // dai < rad
+            // Calculates the needed dart so together with the existing dai in the vat is enough to exit wad amount of DAI tokens
+            dart = _toInt256(toDraw / rate);
+            // This is neeeded due lack of precision. It might need to sum an extra dart wei (for the given DAI wad amount)
+            dart = _mul(uint256(dart), rate) < toDraw ? dart + 1 : dart;
+        }
+    }
+
+    function _getWipeDart(
+        uint256 dai,
+        address urn,
+        bytes32 ilk
+    ) internal view returns (int256 dart) {
+        // Gets actual rate from the vat
+        (, uint256 rate,,,) = vat.ilks(ilk);
+        // Gets actual art value of the urn
+        (, uint256 art) = vat.urns(ilk, urn);
+
+        // Uses the whole dai balance in the vat to reduce the debt
+        dart = _toInt256(dai / rate);
+        // Checks the calculated dart is not higher than urn.art (total debt), otherwise uses its value
+        dart = uint256(dart) <= art ? - dart : - _toInt256(art);
+    }
+
+    function _getWipeAllWad(
+        address usr,
+        address urn,
+        bytes32 ilk
+    ) internal view returns (uint256 wad) {
+        // Gets actual rate from the vat
+        (, uint256 rate,,,) = vat.ilks(ilk);
+        // Gets actual art value of the urn
+        (, uint256 art) = vat.urns(ilk, urn);
+
+        // Gets DAI balance of the urn in the vat
+        uint256 dai = vat.dai(usr);
+
+        // If there was already enough DAI in the vat balance, no need to join more
+        uint256 debt = _mul(art, rate);
+        if (debt > dai) {
+            uint256 rad = _sub(debt, dai);
+            wad = rad / RAY;
+
+            // If the rad precision has some dust, it will need to request for 1 extra wad wei
+            wad = _mul(wad, RAY) < rad ? wad + 1 : wad;
+        }
+    }
+
+    function _open(
+        bytes32 ilk,
+        address usr
+    ) internal returns (uint256 cdp) {
+        cdp = manager.open(ilk, usr);
+    }
+
+    function _flux(
+        uint256 cdp,
+        address dst,
+        uint256 wad
+    ) internal {
+        manager.flux(cdp, dst, wad);
+    }
+
+    function _move(
+        uint256 cdp,
+        address dst,
+        uint256 rad
+    ) internal {
+        manager.move(cdp, dst, rad);
+    }
+
+    function _frob(
+        uint256 cdp,
+        int256 dink,
+        int256 dart
+    ) internal {
+        manager.frob(cdp, dink, dart);
+    }
```

Comment:
The reasoning of leaving this functions as internal is because use can use the main proxy actions public functions?

Reply:
Yes. We can actually call the manager directly instead of the internal wrapper functions, as they are useless. I will change to that.

Reply:
Sounds good.

Reply:
Done.